src/controllers/todoController.js

```diff
@@ -13,49 +13,51 @@ const createTodo = async (req, res) => {
 
     res.status(200).send({ newTodo, message: 'todo created' })
   } catch (e) {
-    res.status(401).send({ message: 'Not fullfilled', error: e.message })
+    res.status(500).send({ message: 'Not fullfilled', error: e.message })
```

Comment:
Try to set as many Logic Error Messages as possible for User

Reply:
Also don't throw server errors directly to client, just show simple user-friendly messages